### PR TITLE
e2e test for best practice content pages

### PR DESCRIPTION
**Pull request by heyitsmebev**
As part of the [end2end epic](https://github.com/GSA/notifications-admin/issues/1540), this issue is to create testing for the new Best Practice content pages side nav

In my implementation, I'm using an endpoint to retrieve feature flags in E2E tests because Playwright runs in a browser environment and can't access Flask's server-side configuration (current_app.config). This endpoint allows Playwright to check feature flags dynamically, enabling conditional tests based on the feature's state.

### CODE REVIEW

File: tests/end_to_end/test_best_practices_content_pages.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+def test_best_practices_side_menu(authenticated_page):
+    page = authenticated_page
+
+    page.goto(f"{E2E_TEST_URI}/best-practices")
+
+    page.wait_for_load_state("domcontentloaded")
+    check_axe_report(page)
+
+    response = page.request.get(f"{E2E_TEST_URI}/test/feature-flags")
```
Comment (ccostino) [inline tests/end_to_end/test_best_practices_content_pages.py:19]: Just double checking my understanding:  this is just getting the response of the feature-flags URI, not actually going to the page, right?

**Review by ccostino — COMMENTED** [review]
Thank you, @heyitsmebev!  I had just one quick question to double check on, everything else looks good to me.

File: tests/end_to_end/test_best_practices_content_pages.py (hunk 1 above)
Comment (heyitsmebev) [reply tests/end_to_end/test_best_practices_content_pages.py:19]: That's correct, my understanding is that the `get` retrieves the response from the server without navigating to the page.

Comment (ccostino) [reply tests/end_to_end/test_best_practices_content_pages.py:19]: Thank you!

**Review by ccostino — APPROVED** [review]
Thanks @heyitsmebev!

**Pull request merged**